prompting/utils/uids.py

```diff
@@ -53,6 +55,8 @@ def get_random_uids(self: BaseNeuron, k: int, exclude: List[int] = None) -> torc
     Notes:
         If `k` is larger than the number of available `uids`, set `k` to the number of available `uids`.
     """
+    if settings.TEST and settings.TEST_MINER_IDS:
+        return np.array(random.sample(settings.TEST_MINER_IDS, min(len(settings.TEST_MINER_IDS), k)))
```

Comment:
I think a better implementation would be to add a new samping_mode called "list" and change the `sample_uids` method to just do something like this...

```py
    def sample_uids(self, params: QueryChatRequest) -> list[int]:
        if params.sampling_mode == "list":
            return params.sampling_list
        if params.sampling_mode == "random":
            uids = get_random_uids(k=params.k, exclude=params.exclude or []).tolist()
            return uids
        if params.sampling_mode == "top_incentive":
            metagraph = self.validator.metagraph
            vpermit_tao_limit = self.validator.config.neuron.vpermit_tao_limit

            top_uids = get_top_incentive_uids(metagraph, k=params.k_miners, vpermit_tao_limit=vpermit_tao_limit)

            return top_uids
```

Reply:
In my opinion this should go into the API codebase rather than here.